automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala

```diff
@@ -184,169 +170,20 @@ class SamApiSpec extends AnyFreeSpec with BillingFixtures with Matchers with Sca
-    //in a GPAlloc'd project, the firecloud service account does not have permission to remove the pet SA
-    // @mbemis
-    "should re-create a pet SA in google even if it still exists in sam" ignore {
-      val user = UserPool.chooseStudent
-
-      //this must use a GPAlloc'd project to avoid deleting the pet for a shared project, which
-      //may have unexpected side effects
-      withCleanBillingProject(user) { projectName =>
-        withCleanUp {
-          val petSaKeyOriginal = Sam.user.petServiceAccountKey(projectName)(user.makeAuthToken())
-          val petSaEmailOriginal = getFieldFromJson(petSaKeyOriginal, "client_email")
-          val petSaKeyIdOriginal = getFieldFromJson(petSaKeyOriginal, "private_key_id")
-          val petSaName = petSaEmailOriginal.split('@').head
-
-          register cleanUp Sam.user.deletePetServiceAccountKey(projectName, petSaKeyIdOriginal)(user.makeAuthToken())
-
-          //act as a rogue process and delete the pet SA without telling sam
-          Await.result(googleIamDAO.removeServiceAccount(GoogleProject(projectName), ServiceAccountName(petSaName)), Duration.Inf)
-
-          val petSaKeyNew = Sam.user.petServiceAccountKey(projectName)(user.makeAuthToken())
-          val petSaEmailNew = getFieldFromJson(petSaKeyNew, "client_email")
-          val petSaKeyIdNew = getFieldFromJson(petSaKeyNew, "private_key_id")
-
-          register cleanUp Sam.user.deletePetServiceAccountKey(projectName, petSaKeyIdNew)(user.makeAuthToken())
-
-          petSaEmailOriginal should equal(petSaEmailNew) //sanity check to make sure the SA is the same
-          petSaKeyIdOriginal should not equal petSaKeyIdNew //make sure we were able to generate a new key and that a new one was returned
-        }
-      }
-    }
-
-    "should get an access token for a user's own pet service account" in {
-      val user = UserPool.chooseStudent
-
-      withCleanBillingProject(UserPool.chooseProjectOwner, List(user.email)) { project =>
-        // get my pet's email
-        val petEmail1 = Sam.user.petServiceAccountEmail(project)(user.makeAuthToken())
-
-        val scopes = Set("https://www.googleapis.com/auth/userinfo.email", "https://www.googleapis.com/auth/userinfo.profile")
-
-        // get my pet's token
-        val petToken = Sam.user.petServiceAccountToken(project, scopes)(user.makeAuthToken())
-
-        // convert string token to an AuthToken
-        val petAuthToken = new AuthToken {
-          override def buildCredential() = ???
-
-          override lazy val value = petToken
-        }
-
-        // get my pet's email using my pet's token
-        val petEmail2 = Sam.user.petServiceAccountEmail(project)(petAuthToken)
-
-        // result should be the same
-        petEmail2 shouldBe petEmail1
-      }
-    }
-
-    "should arbitrarily choose a project to return a pet key for when the user has existing pets" in {
```

Comment:
things doin't work the way this test imply. `arbitraryPetServiceAccountKey` creates its own google project behind the scenes, it does not use an existing project if it exists.